config/itemTypes.json

```
@@ -304,4 +304,40 @@
 }, {
   "id": "SentinelWeapons",
   "name": "Sentinel"
+}, {
+  "id": "Events/RelayReconstruction",
+  "name": "Resource"
+}, {
+  "id": "Items/RelayReconstruction",
+  "name": "Resource"
+}, {
+  "id": "Graphics/CustomUI/Backgrounds",
+  "name": "Theme Background"
+}, {
+  "id": "Graphics/CustomUI/Sounds",
+  "name": "Theme Sound"
+}, {
+  "id": "Graphics/CustomUI",
+  "name": "Themes"
+}, {
+  "id": "NotePacks",
+  "name": "Note Packs"
+}, {
+  "id": "ActionFigureDioramas",
+  "name": "Ship Decoration"
+}, {
+  "id": "PoseSets",
+  "name": "Ship Decoration"
+}, {
+  "id": "Deimos/WoundedInfested",
+  "name": "Pets"
+}, {
+  "id": "LichToken",
+  "name": "Resource"
+}, {
+  "id": "Items/Deimos/Seriglass",
+  "name": "Resource"
+}, {
+  "id": "ZanukaPets/ZanukaPetParts",
+  "name": "PEt Resource"
```

Typo?

---

yep, i'll fix that now

```suggestion
  "name": "Pet Resource"
```